lib/aempartsize.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
```

Comment:
Probably for all copyright headers in new files: should be `2021`.

Reply:
I'll review and fix those